### PR TITLE
Rename: Card content image format class

### PR DESCRIPTION
**Pull request by andygout**
cc @ironsidevsquincy

### CODE REVIEW

File: components/_card.scss [hunk 1]
```diff
@@ -116,7 +116,7 @@ $rule-size: 1px;
 		padding-left: 10px;
 		width: 60%;
 
-		&.card__content--has-image--portrait {
+		&.card__content--has-portrait-image {
 			margin-left: 35%;
 
 			.core & {
```
Comment (ironsidevsquincy) [inline components/_card.scss]: unrelated, but what's this doing? Card seems to break in core mode

File: components/_card.scss [hunk 2]
```diff
@@ -136,7 +136,7 @@ $rule-size: 1px;
 				padding-left: 10px;
 				width: 60%;
 
-				&.card__content--has-image--portrait {
+				&.card__content--has-portrait-image {
 					margin-left: 35%;
 
 					.core & {
```
Comment (ironsidevsquincy) [inline components/_card.scss]: ditto

**Comment by ironsidevsquincy** [comment]
👍

File: components/_card.scss (hunk 1 above)
Comment (andygout) [inline components/_card.scss]: IE8 fix - screengrabs at the bottom of this PR: https://github.com/Financial-Times/n-card/pull/135

**Pull request merged**